app/services/slack_service/group_member_messaging.rb

```diff
@@ -81,6 +81,19 @@ def time_entry_format_warning_message(member, entries)
       }
     end
 
+    def format_desc(entries)
+      entries.map do |entry|
+        formatted_desc = entry.description.split(" ").map do |word|
+          if word.include?("#")
+            word = "`#{word}`"
+          else
+            word
+          end
+        end.join(" ")
+        "* #{formatted_desc} (#{entry.length})"
+      end.join("\n")
+    end
```

Comment:
this is a kind of pseudocode, I haven't tested it, the refactor will be the first step, then try to move `entry.description.gsub(/(#\w+)/) { |label| "`#{label}`" }` to a method

```suggestion
    def format_desc(entries)
      entries.map do |entry|
        formatted_desc = entry.description.gsub(/(#\w+)/) { |label| "`#{label}`" }
        "* #{formatted_desc} (#{entry.length})"
      end.join("\n")
    end
```